CHANGELOG_PENDING.md

```diff
@@ -17,6 +17,7 @@
 ### FEATURES:
 
 ### IMPROVEMENTS:
+- [kms] \#3370 Refactored validator/kms connection (@jleni)
```

Comment:
Nit: we usually use the text inside the [] to mark the package that was impacted (here [privval]).